apps/pie-storybook/package.json

```diff
@@ -3,24 +3,24 @@
   "version": "0.1.0",
   "description": "Storybook for PIE Design System compoents",
   "scripts": {
-    "storybook-dev": "storybook dev -p 6006",
-    "storybook-build": "storybook build --output-dir build",
-    "storybook-serve": "serve --no-request-logging build"
+    "dev": "storybook dev -p 6006",
+    "build": "storybook build --output-dir dist",
+    "storybook-serve": "serve --no-request-logging dist"
```

Comment:
It'd be nice to have the keep the naming consistency for the `storybook-serve` script.

Reply:
Have updated to `serve`